scripts/hostcfgd

```diff
@@ -759,8 +771,12 @@ class AaaCfg(object):
 
     def modify_single_file(self, filename, operations=None):
         if operations:
-            cmd = "sed -e {0} {1} > {1}.new; mv -f {1} {1}.old; mv -f {1}.new {1}".format(' -e '.join(operations), filename)
-            os.system(cmd)
+            e_operations_str = ' -e '.join(operations)
+            e_operations_list = split(e_operations_str)
```

Comment:
>e_operations_list [](http://example.com/codeflow?start=12&length=17)

operations is already a list, no need to join and then split.

Reply:
I think the list is joined by `' -e '`, not whitespace?

Reply:
I got it. Better to implement the transformation on a list directly.